parachain/pallets/ethereum-beacon-client/src/config/mainnet.rs

```diff
@@ -0,0 +1,3 @@
+pub const SLOTS_PER_EPOCH: u64 = 32;
+pub const EPOCHS_PER_SYNC_COMMITTEE_PERIOD: u64 = 256;
```

Comment:
Can we instead of `mainnet` or `minimal` name these files the same as the runtime they will be included in. `snowbase`, `snowblink`, `snowbridge`?